src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -326,4 +330,12 @@ private void addScmDetails() {
             return Collections.emptyList();
         }
     }
+
+    private String getJobDisplayName() {
+        if (job.getParent() instanceof MultiBranchProject) {
```

Comment:
@jenkinsci/code-reviewers anyone aware of a better way to determine project type without pulling in branch API plugin?

Reply:
Wouldn't `getFullDisplayName` be better in all cases? This looks like a hack to me, and doesn't handle normal folders anyway.

Reply:
well, I considered getFullDisplayName but that can also become some very long names especially when you start considering folders.
So it's either full display name or somewhat more useful display names, give somewhat useful names.

Reply:
Well, up to the users configuring their Jenkins instance how they want display names to work.

You wouldn't forcibly limit what gets shown to 20 chars without a good reason either, would you? May just not be good enough to distinguish one project from another. Similarly here.

Reply:
As a trivial example, consider two GitHub org folders (say, `acme` and `coyote-inc`, both having a repo called `site`, and both having a branch `master`.

`acme » site » master` and `coyote-inc » site » master` are the only reasonable representations.

Reply:
Good point 👍 Thanks for the feedback 😅